File: docs/code-examples/scalar_simple.rs
//! Log a scalar over time.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_scalar").save("/tmp/kek.rrd")?;

    for step in 0..1_000_000 {
        rec.set_time_sequence("step", step);
        rec.log(
            "scalar",
            &rerun::TimeSeriesScalar::new((step as f64 / 10.0).sin()),
        )?;
    }

    Ok(())
}

File: docs/code-examples/scalar_simple.py
"""Log a scalar over time."""
import math

import rerun as rr

rr.init("rerun_example_scalar")
rr.save("/tmp/scalars.rrd")

for step in range(0, 64000):
    rr.set_time_sequence("step", step)
    rr.log("scalar", rr.TimeSeriesScalar(math.sin(step / 10.0)))
